src/commands/radio/search.js

```diff
@@ -0,0 +1,54 @@
+const { SearchCommand, CommandError, SwitchbladeEmbed } = require('../../')
+
+module.exports = class TuneIn extends SearchCommand {
+  constructor (client) {
+    super(client, {
+      name: 'tunein',
+      category: 'radio',
+      parameters: [
+        {
+          type: 'string',
+          full: true
+        }
+      ],
+      requirements: { apis: ['tunein'] }
+    })
+  }
+
+  async search (_context, keyword) {
+    const results = await this.client.apis.tunein.search(keyword)
+    return results
+  }
+
+  searchResultFormatter (radio) {
+    return `${this.formatTitle(radio.text)}`
+  }
+
+  async handleResult ({ t, channel, author, language }, radio) {
+    this.getRadioDescription(radio.now_playing_id)
+      .then(r => {
+        const song = r.current_song ? `${r.current_artist} - ${r.current_song}` : 'N/A'
+        channel.send(
+          new SwitchbladeEmbed(author)
+            .setTitle(this.formatTitle(r.title || r.name))
+            .addField('Now Playing:', song, true)
+            .addField('Genre:', r.genre_name, true)
```

Comment:
These should be added to the `commands.json` file as strings instead of being hardcoded. Check other PRs to see how our localization system works :)